examples/firebase-functions/README.md

```diff
@@ -0,0 +1,34 @@
+# Firebase function example
+
+Basic example using tinyhttp, typescript with firebase function.
+
+Feel free to modify `.firebaserc` and `firebase.json` to fit your needs.
+
+## Install and login with firebase cli tool
```

Comment:
could you please split this into two subsections of one section?

e.g.

## Firebase CLI Setup

### Install

```
// ...
```

### Login

```
// ...
```